spy/libspy/src/libc.c

```diff
@@ -1,6 +1,7 @@
 #include "spy.h"
 
-int memcmp(const void *s1, const void *s2, size_t n) {
+int
+memcmp(const void *s1, const void *s2, size_t n) {
```

Comment:
oh, this file is a leftover from a distant past in which I was trying to compile for barebone wasm. We should delete it